tests/test_invoices.py

```diff
@@ -599,6 +599,13 @@ def test_autocleaninvoice(node_factory):
     assert len(l1.rpc.listinvoices('inv1')['invoices']) == 0
     assert len(l1.rpc.listinvoices('inv2')['invoices']) == 0
 
+    # stress test
+    l1.rpc.autocleaninvoice(cycle_seconds=0)
+    l1.rpc.autocleaninvoice(cycle_seconds=1)
+    l1.rpc.autocleaninvoice(cycle_seconds=0)
```

Comment:
You should put `@pytest.mark.xfail(strict=True)` at the top of this test, and remove it in the final commit.  That avoids breaking bisect...